tstest/chonktest/tailchonk_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package chonktest
+
+import (
+	"testing"
+
+	"tailscale.com/tka"
+)
+
+func TestImplementsChonk(t *testing.T) {
+	for _, tt := range []struct {
+		name     string
+		newChonk func(t *testing.T) tka.Chonk
+	}{
+		{
+			name:     "Mem",
+			newChonk: func(t *testing.T) tka.Chonk { return &tka.Mem{} },
+		},
+		{
+			name:     "FS",
+			newChonk: func(t *testing.T) tka.Chonk { return &tka.FS{Base: t.TempDir()} },
+		},
+	} {
+		RunChonkTests(t, tt.name, func(t *testing.T) tka.Chonk { return tt.newChonk(t) })
```

Comment:
this tt.name you pass is purely for the callee to do the subtest? But you could do that yourself here and have one fewer parameter.

Reply:
Oh neat, I didn't realise you could nest `t.Run()`.